src/businessLogic/orderLogic.js

```diff
@@ -0,0 +1,20 @@
+const pizzaOrderMapper = (pizza, amount = 1) => ({ name: pizza.name, size: pizza.size, cost: pizza.cost, amount })
+const identifier = pizza => pizza.name + pizza.size
+const sortByName = (a, b) => identifier(a) > identifier(b) ? 1 : (identifier(b) > identifier(a) ? -1 : 0)
+
+export const addPizza = (order, pizza) => {
```

Comment:
What is your thoughts on that this way of doing code can quickly become difficult to read?
Especially if someone is not used to this coding style

Reply:
Yeah, agreed.

This is ugly. Basically, is that I want to use the name + size as primary key for pizzas in an order.
However, I could resolve this into a property named "ID" in my pizzaOrderMapper-function instead.

Reply:
This comment is enough, no need to change :D 
Thanks for sharing